cli/command/cluster/add.go

```diff
@@ -71,9 +81,9 @@ func NewAddCommand(curveadm *cli.CurveAdm) *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringVarP(&options.descriotion, "description", "m", "", "Description for cluster")
+	flags.StringVarP(&options.description, "description", "m", "", "Description for cluster")
 	flags.StringVarP(&options.filename, "topology", "f", "", "Specify the path of topology file")
-
+	flags.StringVarP(&options.deployType, "deploy-type", "t", "develop", "Specify the deploy type of cluster")
```

Comment:
suggestion: convert `deploy-type` to `type`.

Reply:
I see,I'll fix this later.